### PR TITLE
add HiddenInput element

### PR DESCRIPTION
**Pull request by mate-code**

### CODE REVIEW

**Comment by jobrachem** [comment]
Hi Marius, thanks for the PR! Do you have a little test-experiment for me to check whether everything works?

**Comment by mate-code** [comment]
Hi Johannes,
I added another commit that adds HiddenInput to __init__.py

Minimal example:
```python
import alfred3 as al
exp = al.Experiment()

@exp.member
class Demo(al.Page):

    def on_exp_access(self):
        self += al.HiddenInput(name="hidden_demo")
```

**Comment by jobrachem** [comment]
Hi @mate-code, thanks for providing a minimal example. Running this example on your PR leads to the following error:

```
Traceback (most recent call last):
  File "/Users/johannesbrachem/Documents/code/py/202204081148-hiddeninput/script.py", line 1, in <module>
    import alfred3 as al
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/__init__.py", line 4, in <module>
    from .experiment import Experiment
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/experiment.py", line 54, in <module>
    from .data_manager import DataManager
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/data_manager.py", line 30, in <module>
    from .util import flatten_dict
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/util.py", line 21, in <module>
    from .element.display import Label
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/element/display.py", line 26, in <module>
    from .input import SingleChoiceButtons
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/element/input.py", line 2057, in <module>
    class HiddenInput(InputElement):
  File "/Users/johannesbrachem/Documents/git/alfred/src/alfred3/element/input.py", line 2079, in HiddenInput
    base_template = jinja_env.get_template("EmptyElement.html.j2")
  File "/Users/johannesbrachem/.pyenv/versions/3.10.3/envs/test/lib/python3.10/site-packages/jinja2/environment.py", line 1000, in get_template
    return self._load_template(name, globals)
  File "/Users/johannesbrachem/.pyenv/versions/3.10.3/envs/test/lib/python3.10/site-packages/jinja2/environment.py", line 959, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/Users/johannesbrachem/.pyenv/versions/3.10.3/envs/test/lib/python3.10/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/johannesbrachem/.pyenv/versions/3.10.3/envs/test/lib/python3.10/site-packages/jinja2/loaders.py", line 344, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: EmptyElement.html.j2
```

I made a comment in your PR to show how to fixt the issue, it's just a wrong filepath for the template.

File: src/alfred3/element/input.py [hunk 1]
```diff
@@ -2051,3 +2051,30 @@ def prepare_web_widget(self):
...
+            class Demo(al.Page):
+                name = "demo"
+
+                def on_exp_access(self):
+                    self += al.HiddenInput(name="hi1", default="fixed")
+
+    """
+
+    base_template = jinja_env.get_template("EmptyElement.html.j2")
+    element_template = jinja_env.get_template("HiddenInputElement.html.j2")
```
Comment (jobrachem) [inline src/alfred3/element/input.py:2080]: For the filepaths here, you need to include the full relative path. The `jinja_env` points to the directory `templates`, so you need to use, for example, `html/EmptyElement.html.j2`.

Comment (mate-code) [reply src/alfred3/element/input.py]: Done in commit c50a586eda2b42666af1e15ea10350b87ff18989

Comment (jobrachem) [reply src/alfred3/element/input.py]: Nice! :)

**Pull request merged**